README.md

```diff
@@ -1,189 +1,116 @@
-	return storageClient.storageAccounts.update(resourceGroupName, storageAccountName, updateParameters, callback);
-
-Note that modifying the SKU for a production storage account may have associated costs. For example, if you convert a locally redundant storage account to a geo-redundant storage account, you will be charged for replicating your data to the secondary region. Before you modify the SKU for a production account, be sure to consider any cost implications. See [Azure Storage replication](https://azure.microsoft.com/documentation/articles/storage-redundancy/) for additional information about storage replication.
-
-### Check storage account name availability
-
-The sample checks whether a given storage account name is available in Azure: 
-
-	console.log('\n-->Checking if the storage account name : ' + storageAccountName + ' is available.');
-	return storageClient.storageAccounts.checkNameAvailability(storageAccountName, callback);
-
-### Delete the storage account and resource group
-
-Running cleanup.js deletes the storage account that the sample created:
-
-	console.log('\nDeleting storage account : ' + storageAccountName);
-	return storageClient.storageAccounts.deleteMethod(resourceGroupName, storageAccountName, callback);
-
-It also deletes the resource group that the sample created:
-
-	console.log('\nDeleting resource group: ' + resourceGroupName);
-	return resourceClient.resourceGroups.deleteMethod(resourceGroupName, callback);
-
-
-## More information
-
-- [Azure SDK for Node.js](https://github.com/Azure/azure-sdk-for-node) 
-- [Azure Storage Documentation](https://azure.microsoft.com/services/storage/)
\ No newline at end of file
+# Official Microsoft Sample
+
+<!-- 
+Guidelines on README format: https://review.docs.microsoft.com/help/onboard/admin/samples/concepts/readme-template?branch=master
+
+Guidance on onboarding samples to docs.microsoft.com/samples: https://review.docs.microsoft.com/help/onboard/admin/samples/process/onboarding?branch=master
+
+Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
+-->
+
+A sample code to create or update keyvault and create or update secret in a keyvault.
+
+## Contents
+
+| File/folder       | Description                                |
+|-------------------|--------------------------------------------|
+| `index.js`        | Sample source code.                        |
+| `.gitignore`      | Define what to ignore at commit time.      |
+| `package.json`    | Define dependencies.                       |
+| `README.md`       | This README file.                          |
+| `LICENSE`         | The license for the sample.                |
+
+## Prerequisites
+
+Refer to this azure stack doc for prerequisites (link)[https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profile-nodejs].

```

Comment:
change this to ```[link](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profile-nodejs)``` to create a link in markdown

Reply:
I will update it to remove the attempted formatting to be consistent with other links that are not formatted. I think this part was initially copy-pasted from Azure sample repo, so I didn't notice this issue.